narratives/ncov_sit-rep_2020-04-10.md

```diff
@@ -0,0 +1,270 @@
+<br><br>
+During this early period, the majority of sampled and inferred ancestral cases were located in Asia.
+This is consistent with circulation largely within Asia for the first month of the outbreak.  
+
+<!-- There is no right side text -->
+
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text -->
+# [Initial introductions to other regions](https://nextstrain.org/ncov/global/2020-04-09?dmax=2020-02-15&dmin=2020-01-15&p=full&d=map,tree)
+
+During the 2nd month of the outbreak, we start to see individual cases crop up in North America, Europe and Oceania.
+These early samples still group with samples from Asia, and generally lack the hallmarks of local transmission.
+<br><br>
+This is consistent with initial sparks being thrown from the main outbreak in Asia to other parts of the world. Although most of these sparks did not result in localized outbreaks, at least two of these introductions took off.
+<!-- There is no right side text -->
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text -->
+# [Cryptic transmission in Europe from mid-January](https://nextstrain.org/ncov/global/2020-04-09?d=tree,map&dmax=2020-02-14&label=clade:A2&p=full)
+
+Zooming in to the top of the tree for the same time period, we see an ancestral European virus (internal node) that seeded the large European outbreak.
+<br><br>
+This indicates that while we hadn't yet sequenced any samples from this clade, the origins of the European outbreak were already cryptically circulating by mid-January.
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text -->
+# [Cryptic transmission in North America from mid-January](https://nextstrain.org/ncov/global/2020-04-09?d=tree,map&dmax=2020-02-14&label=clade:B1&p=full)
+
+Similarly, zooming in to the bottom of the tree, we see an ancestral North American virus (internal node) that initiated what would later become a large outbreak in North America.
+<br><br>
+This indicates that although we only had 1 sequenced case from this cluster at the time, the virus was circulating cryptically within North America by mid-January.
+
+<!-- There is no right side text -->
+
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text -->
+# [Expansion of North American and European outbreaks](https://nextstrain.org/ncov/global/2020-04-09?dmax=2020-03-15&dmin=2020-02-15&p=full&d=tree,map)
+
+During the 3rd month of the outbreak, these two clusters expanded to cause large outbreaks within Europe and North America, respectively.
+<!-- There is no right side text -->
+
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text -->
+# [Spread across international borders](https://nextstrain.org/ncov/global/2020-04-09?dmax=2020-03-15&dmin=2020-02-15&f_region=North%20America&p=full&r=division&d=tree,map)
```

Comment:
It's unfortunate that the map doesn't zoom appropritately here. I'm guessing this is due to the map being shown in "triplicate" for nCoV.

Reply:
I suspect that coloring by division would better illustrate your point. With just the red coloring by country, there could be state level structure that we just don't see due to the color choice.